src/main/webapp/assets/modules/common/vnwConfigService.js

```diff
@@ -775,7 +775,7 @@ techlooper.factory("vnwConfigService", function (jsonValue, $translate, $rootSco
 
     locationsSelectize: {
       items: locations.map(function (location) {
-        return {id: location.location_id, translate: location[vnwLang]};
+        return {id: location.location_id, translate: location[vnwLang], en: location.lang_en};
```

Comment:
Line is too long.<br>Identifier 'location_id' is not in camel case.<br>Identifier 'lang_en' is not in camel case.